src/EditorFeatures/CSharpTest/Diagnostics/Suppression/SuppressionTests.cs

```diff
@@ -121,8 +160,8 @@ void Method()
     {{
 #pragma warning disable CS0219 // {CSharpResources.WRN_UnreferencedVarAssg_Title}
         int x = 0
-#pragma warning restore CS0219 // {CSharpResources.WRN_UnreferencedVarAssg_Title}
               + 1;
+#pragma warning restore CS0219 // {CSharpResources.WRN_UnreferencedVarAssg_Title}
```

Comment:
This was the only change in behaviour, but it seemed reasonable to me

Reply:
Is there already a text that has something like: 

```c#
int x = 0,
     y = 0;
```

and do we car3e if it disables the warning for both?

Reply:
This aligns with [@Youssef1313s comment](https://github.com/dotnet/roslyn/pull/56285#pullrequestreview-750620102), wonder how much it matters

Reply:
hmm.. good point. With these changes, this applies the pragma to the whole statement - ie, both variable declarators.

Personally I don't think its an issue, but I could be convinced otherwise.